smrf/distribute/thermal.py

```diff
@@ -229,8 +234,8 @@ def initialize(self, topo, data):
         """
 
         self._logger.debug('Initializing distribute.thermal')
-        if self.gridded:
-            self._initialize(topo, data.metadata)
+        # if self.gridded:
```

Comment:
Same here?

Reply:
We were only initializing if it was gridded but now it's done each time to get the base thread variables. Cleaned up